<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerDialog</class>
 <widget class="QDialog" name="ManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 125, 173);</string>
  </property>
  <widget class="QPushButton" name="Button4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonReturn">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>240</y>
     <width>122</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button8">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>400</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPageDown">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>400</y>
     <width>122</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>下翻</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>480</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPoint">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>480</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button7">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>400</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPageUp">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>320</y>
     <width>122</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>上翻</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>二</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button9">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>400</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonEnter">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>480</y>
     <width>122</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button0">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>560</y>
     <width>121</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditPwd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>421</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>ManagerDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>612</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
